quesma/clickhouse/clickhouse.go

```diff
@@ -498,11 +489,8 @@ func (lm *LogManager) BuildInsertJson(tableName string, data types.JSON, inValid
 		return js, nil, nil
 	}
 	var attrsMap map[string][]interface{}
-	var othersMap SchemaMap
 	if len(config.attributes) > 0 {
-		attrsMap, othersMap, _ = BuildAttrsMapAndOthers(mDiff, config)
-	} else if config.hasOthers {
-		othersMap = mDiff
+		attrsMap, _ = BuildAttrsMapAndOthers(mDiff, config)
```

Comment:
Nit: you could rename `BuildAttrsMapAndOthers` to `BuildAttrsMap`

Reply:
Right, thx